src/ai-bundle/config/options.php

```diff
@@ -162,6 +162,14 @@
                                     ->info('Include tool definitions at the end of the system prompt')
                                     ->defaultFalse()
                                 ->end()
+                                ->booleanNode('enable_translation')
+                                    ->info('Enable translation for the system prompt')
+                                    ->defaultFalse()
+                                ->end()
+                                ->scalarNode('translation_domain')
+                                    ->info('The translation domain for the system prompt')
+                                    ->defaultNull()
```

Comment:
let's use the default domain as default here - makes it easier to get started:
```suggestion
                                    ->defaultValue('messages')
```

Reply:
Exactly. Passing in `messages` here makes assumptions about what the translator dictates as the default domain. It would make us "know" about something outside of our concern.

Reply:
null is fine